apps/condo/domains/organization/access/Organization.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Generated by `createschema organization.Organization 'country:Select:ru,en; name:Text; description?:Text; avatar?:File; meta:Json; employees:Relationship:OrganizationEmployee:CASCADE; statusTransitions:Json; defaultEmployeeRoleStatusTransitions:Json' --force`
+ */
+
+const { getByCondition } = require('@core/keystone/schema')
+
+async function canReadOrganizations ({ authentication: { item: user } }) {
+    if (!user) return false
+    if (user.isAdmin) return {}
+    return {
+        employees_some: { user: { id: user.id } },
+    }
+}
+
+async function canManageOrganizations ({ authentication: { item: user } }) {
+    if (!user) return false
+    if (user.isAdmin) return true
+    return {
+        // user is inside employee list
+        employees_some: { user: { id: user.id, canManageOrganization: true } },
+    }
+}
+
+async function checkOrganizationPermission (userId, organizationId, permission) {
```

Comment:
it's uncommon to export something instead of permission checks

Reply:
Moved `checkOrganizationPermission` to organization/utils/accessSchema